src/components/SearchBox.vue

```diff
@@ -2,7 +2,27 @@
   <form role="search" action="" @submit.prevent="onFormSubmit">
     <slot>
       <ais-input :search-store="searchStore" :placeholder="placeholder" :autofocus="autofocus"></ais-input>
-      <button type="submit" :class="bem('submit')">
+      <div v-if="showLoadingIndicator" :style="searchStore.isSearchStalled ? 'display:block;' : 'display:none;'" :class="bem('loading-indicator')" >
```

Comment:
You should use the `hidden` attribute, which implies `display: none;` but is better for a11y

Reply:
`:hidden="!searchStore.isSearchStalled"`